src/js/libcs/RenderBackends.js

```diff
@@ -868,3 +868,9 @@ globalInstance.exportPDF = function() {
         exportWith(PdfWriterContext, wnd);
     });
 };
+
+globalInstance.exportPNG = function() {
+    var data = csctx.canvas.toDataURL();
+    var wnd = window.open('about:blank', '_blank');
+    wnd.location.href = data;
```

Comment:
When I implemented PNG export for ComplexCurves I found it preferable to generate an `<a>` element pointing to the data URL then invoking its `click` event. Unlike opening a new window, this also works with popup blockers.

Moreover, you can specify a file name in the data URL, too.